packages/extension-chakra-storefront/src/components/toaster/index.jsx

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2025, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import React from 'react'
+import {
+  Toaster as ChakraToaster,
+  Portal,
+  Stack,
+  Toast,
+  createToaster
+} from "@chakra-ui/react"
+
+// A toaster is a shared global instance that can be used to create and manage toasts.
+export const toaster = createToaster({
+  placement: "top-end"
+})
+
+export default function Toaster({ toaster }) {
```

Comment:
Any way to customise toaster look and feel from here?

Reply:
We've been using the default Chakra v2 styles. However, Chakra v3 no longer ships default styles for toasts and now it looks a little bit different but still does its job and I think it looks better now. 

It's not my goal to re-style it back to what it looked like in v2.